pkgs/by-name/li/lightning-terminal/package.nix

```diff
@@ -101,6 +155,5 @@ buildGoModule rec {
     changelog = "https://github.com/lightninglabs/lightning-terminal/releases/tag/v${version}";
     maintainers = with lib.maintainers; [ HannahMR ];
     mainProgram = "litcli";
```

Comment:
By the way, why the main program is `litcli`? The main daemon is `litd` and `litcli` is a CLI tool to manage it.